src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/repo.py

```diff
@@ -75,16 +75,12 @@ def create_update(self):
             s3_key_path=None,
             account_id=DEPLOYMENT_ACCOUNT_ID,
         )
-        # Update the stack if the repo and the ADF controlled stack exist,
-        # return if the repo exists but no stack (previously made)
+
         _repo_exists = self.repo_exists()
         _stack_exists = cloudformation.get_stack_status()
         if _repo_exists and not _stack_exists:
+            # return when the repository exists without a stack (previously made)
             return
-        if not _repo_exists and not _stack_exists:
-            LOGGER.info(
-                'Ensuring State for CodeCommit Repository Stack %s on Account %s',
-                self.name,
-                self.account_id,
-            )
-            cloudformation.create_stack()
```

Comment:
The change here will perform updates to the CodeCommit CloudFormation stack.
Before we only created the repository, we didn't update it. I feel we should protect the code from being deleted by accident.
So we will also need an update policy to retain the repository just in case for that to be safe.
Can you add that?

See https://github.com/awslabs/aws-deployment-framework/blob/9420cdb908287dc545eff5a5bb3018f51b091046/src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/templates/codecommit.yml

Reply:
Sure added it in [f70bb46](https://github.com/awslabs/aws-deployment-framework/pull/469/commits/f70bb46d1719d21e1b7a0766d4b082948f25590b)